### ISSUE TITLE
Original DELF model by authors has BN for spatial attention

### ISSUE THREAD

**Issue by dgominski** [nashory/DeLF-pytorch#36]
See here the definition of the model in Tensorflow by authors :
https://github.com/tensorflow/models/blob/master/research/delf/delf/python/training/model/delf_model.py

They use BatchNorm between conv1 and conv2 of the spatial attention module.
